src/lib/settings/settings.py

```diff
@@ -137,8 +171,93 @@ def _get_default_metrics_extractor_role_arn(self, account_id: str) -> str:
         return f"arn:aws:iam::{account_id}:role/role-salmon-cross-account-extract-metrics-dev"
 
     def _process_monitoring_groups(self):
-        # TODO: add wildcards replacement for all the resource types (glue, lambda, etc.)
-        pass
+        # Get resource names dict
+        resource_names = self._get_all_resource_names()
```

Comment:
Looks like you can move resource_names retrieving into  _replace_wildcards and pass into that method only m_env, m_res.

Reply:
Good notice, but in such a case we would need to read glue jobs, lambdas for an account in each monitoring group (which can be 10 groups retrieving the same list of jobs). That's why preparing dictionary only once in the beginning

Reply:
By the way noticed misleading names - m_env should be m_grp, will change :)